manifests/monitor-cf-v2.yml

```diff
@@ -20,6 +20,9 @@ meta:
     cert: (( vault meta.cf_exodus_src "loggregator_tls_rlp_cert" ))
     key:  (( vault meta.cf_exodus_src "loggregator_tls_rlp_key" ))
 
+  environment:                (( grab genesis.env ))
```

Comment:
Move this meta into prometheus.yml as they are common to a lot of manifest fragments